src/main/java/com/xebialabs/overthere/gcp/GcpKeyManager.java

```diff
@@ -0,0 +1,8 @@
+package com.xebialabs.overthere.gcp;
+
+interface GcpKeyManager {
+
+    GcpKeyManager init();
```

Comment:
It is a good practice to add javadocs for interfaces.
Can you please describe interface and each method?